### ISSUE TITLE
_transient_coyote_api_error_count isn't cleared when succesfully creating a profile

### ISSUE THREAD

**Issue by robbinrietdijk** [coyote-team/coyote-wp#120]
When testing the plugin settings with incomplete or incorrect credentials the _transient_coyote_api_error_count increments in the database. When a valid profile is linked I assume the transient should be cleared?

**Comment by jkva** [comment]
Yes. Any successful API request should clear the error counter.
